### ISSUE TITLE
[website] update maven build information

### ISSUE THREAD

**Issue by JanKoehnlein** [eclipse/xtext#1102]
The information on CI with Xtext in the documentation stops at Xtext 2.9.1
It would be helpful to update it to 2.10 and 2.11, especially the section on version numbers.

**Comment by cdietrich** [comment]
@JanKoehnlein i will provide an update

**Comment by cdietrich** [comment]
p.s.: there is another issue somewhere as well. we have to many places for issues :(

**Comment by cdietrich** [comment]
is partially blocked by https://github.com/xtext/maven-xtext-example/pull/43 which is open since 20 days and nobody cares

**Comment by cdietrich** [comment]
does anybody know how this version table is filled?

**Comment by cdietrich** [comment]
would still be interested to find out how this table is filled. @szarnekow @svenefftinge  can you give any hint.
if not i propose to remove the table

**Comment by svenefftinge** [comment]
I don't know.

**Comment by szarnekow** [comment]
I don't have a clue. I'm in favor of removing the table and adding a link to mvn central

**Comment by cdietrich** [comment]
we have updated the table
